general/channels/types/pushinho/utils.py

```diff
@@ -0,0 +1,50 @@
+import boto3
+
+from django.conf import settings
+
+from temba.channels.models import Channel
+from temba.utils.s3 import public_file_storage
+
+
+def upload_icon_to_aws(icon):
+    # Initialize connection with AWS and Boto S3
+    session = boto3.session.Session(
+        aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=settings.AWS_SECRET_ACCESS_KEY,
+    )
+    s3 = session.resource("s3")
+
+    # Upload Icon to bucket on AWS
+    main_icon = icon
+    icon_path = settings.PUSHINHO_ICONS_AWS_PATH + icon.name
+    s3.Bucket(settings.AWS_STORAGE_BUCKET_NAME).put_object(
+        Key=icon_path, Body=main_icon
+    )
+    icon_url = public_file_storage.url(icon_path)
+    return icon_url
+
+
+def create_config(main_icon_url, chat_icon_url, data):
+
+    auto_open = str(data.get("auto_open")).casefold()
+
+    # Dict Config for save informations and show on HTML Returns
+    config = {
+        "main_icon_url": main_icon_url,
```

Comment:
I suggest using constant that lives in the type instance, instead of hardcoded strings.